### ISSUE TITLE
Update GUI for transmission

### ISSUE THREAD

**Issue by scottmc** [haikuports/haikuports#428]
During Google Code-In 2013 a student, Gleb Posobin, wrote a GUI interface for transmission.  This has not been looked at much since.  Need someone to review this and see if there is anything useful here, and if so then to add it into our transmission port.
--------------------------------
Note from old Haikuports site:

Transmission includes an RPC for coomunicating with the transmission-daemon.  http://trac.transmissionbt.com/wiki/rpc this could be used by non c or c++ devs to create a more native GUI 
comment:4 follow-up: ↓ 5 Changed 3 years ago by scottmc

Edit
Reply
Need to update transmission to latest released version, plus add a bep to build latest svn version. GCI2012 student Gleb Posobin work on this task to re-add the GUI to transmission:
 http://www.google-melange.com/gci/task/view/google/gci2012/7953205
It needs to be fully reviewed and applied if ok, and eventually upstreamed. Any takers?
The work is attached to the GCI task page and includes .beps for 2.75 and a diff for the GUI changes.
comment:5 in reply to: ↑ 4 Changed 3 years ago by richienyhus

Edit
Reply
Keywords GCI patch added
Quoting  Jordan:
This is pretty awesome, but I'm not sure what you intend here -- are you wanting to maintain this over the long term?
I don't even know how we'd build binaries for new releases.
IMO it would be better to keep this on a third-party page as the Transmission-Qt for Windows project does on ​ http://sourceforge.net/projects/trqtw/
That said, I'm very happy to work with you, just as I've worked with rb07 from trqtw, on any compatibility issues in libtransmission.
Thanks for writing this!

Diff:  https://www.dropbox.com/s/aledrn5i352fb5v/transmission_r13780_haiku_gui_2.diff
back up:  https://gist.github.com/richienyhus/5182192
By student Gleb Posobin

**Comment by miqlas** [comment]
Updated patch for Transmission 2.92: https://gist.github.com/miqlas/b02a79b05cdfaf0deb4c625c2bda1bc5

Just update, not working yet. Help required.

**Comment by korli** [comment]
Qt GUI enabled in c6b8de94436047fb814ff98f710cd1efe2329392.

**Comment by miqlas** [comment]
Yep, Qt GUI works ok, one can deactivate all the unneded GUI elements like toolbar, status bar, so it can have a really minimal GUI.
But it seems it is also affected by QtHP bug, as it crashes at exit (Second issue here: https://github.com/threedeyes/qthaikuplugins/issues/1 ). Maybe somebody can look into it.
